src/cmd/compile/internal/syntax/token_string.go

```diff
@@ -4,9 +4,9 @@ package syntax
 
 import "strconv"
 
-const _token_name = "EOFnameliteralopop=opop=:=<-*([{)]},;:....breakcasechanconstcontinuedefaultdeferelsefallthroughforfuncgogotoifimportinterfacemappackagerangereturnselectstructswitchtypevar"
+const _token_name = "EOFnameliteralopop=opop=:=<-*([{)]},;:....breakcasechanconstcontinuedefaultdeferelsefallthroughforfuncgogototxnifimportinterfacemappackagerangereturnselectstructswitchtypevar"
```

Comment:
This file is actually auto-generated. So this can't be changed. At the top of the file, you can see the comment
`// Code generated by "stringer -type token -linecomment"; DO NOT EDIT.`

Reply:
Is `tokens_string.go` autogenerated from `tokens.go`? If so will maintaining the order in tokens.go - 
https://github.com/jerrinsg/go-pmem/pull/10/commits/161ea731c48abdbd56c63c40aa70bbcb28a97c47#diff-dcf8ce78525d3af702e737687bc437caR56 help autogenerate this file correctly?

Reply:
Yes. I see the order in tokens_string.go comes from the order in tokens.go. I'll fix this in the next commit. 
Thanks!